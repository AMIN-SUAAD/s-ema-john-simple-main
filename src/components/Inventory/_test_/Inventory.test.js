import React from 'react';
import Inventory from '../Inventory';

import renderer from "react-test-renderer";

it("matches snapshot", () => {
    const tree = renderer.create(<Inventory></Inventory>).toJSON();
    expect(tree).toMatchSnapshot();
})