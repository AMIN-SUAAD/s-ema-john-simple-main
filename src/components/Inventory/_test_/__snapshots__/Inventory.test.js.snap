// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h1>
    Inventory Coming Soon!
  </h1>
</div>
`;
